packages/trpc/server/routers/viewer/eventTypes/util.ts

```diff
@@ -234,11 +235,12 @@ export async function addWeightAdjustmentToNewHosts({
     }));
   }
 
-  const ongoingHostBookings = await BookingRepository.getAllBookingsForRoundRobin({
+  const ongoingHostBookings = await BookingRepository.getBookingsForRoundRobin({
     eventTypeId,
     users: ongoingRRHosts.map((host) => {
       return { id: host.user.id, email: host.user.email };
     }),
+    fetchFromPastDays: FETCH_BOOKINGS_FOR_RR_FROM_PAST_DAYS,
```

Comment:
I don't understand the reason for fetching 28-day booking here. This is the `addWeightAdjustmentToNewHosts` utility function to add weight to new hosts, and it's used in the event type update handler

Reply:
Not sure wdym? Since we are only considering 28 days when handling weights, wouldn't it also make sense to do it the same way when adjusting weights for change in number of hosts?

Reply:
new host also can already have bookings assigned before and  we are limiting to fetch

Reply:
cc @CarinaWolli